include/TPP/Transforms.h

```diff
@@ -59,6 +60,11 @@ FailureOr<linalg::GenericOp> packMatmulOp(RewriterBase &rewriter,
                                           linalg::MatmulOp linalgOp,
                                           ArrayRef<OpFoldResult> tiles);
 
+// Attempt to block a BatchMatmulOp.
```

Comment:
The name implies `Matmul`, the comment implies `BatchMatmul`.

Reply:
yes, it is an overloading to share the same `PackMatmulImpl` driver. Should we call it `packMatmulAndBatchMatmulOp`?